RadixWallet/Features/AssetsFeature/Components/HelperViews/ResourceBalance/ResourceBalanceView.swift

```diff
@@ -241,8 +241,8 @@ extension ResourceBalanceView {
 			VStack(alignment: .leading, spacing: .medium3) {
 				FungibleView(
 					thumbnail: .lsu(viewState.icon),
-					caption1: viewState.title,
-					caption2: viewState.validatorName,
+					caption1: viewState.title ?? "-",
+					caption2: viewState.validatorName ?? "-",
```

Comment:
We don't want to always default to `"-"`. In some cases (e.g., Fungibles), we want to hide the subtitle instead of defaulting to `"-"`; in others (e.g., LSU, Pool Unit), we want to display the subtitle every time and default to `"-"` if the info is missing. I decided to delegate this default logic to the outer views and keep `CaptionedThumbnailView` as configurable as possible.

Reply:
I see, thanks for explanation!